src/csharp/Microsoft.Spark/Utils/CommandSerDe.cs

```diff
@@ -217,11 +217,65 @@ private sealed class UdfWrapperData
             }
         }
 
-        internal static T Deserialize<T>(
+        internal static object DeserializeArrowOrDataFrameUdf(
             Stream stream,
             out SerializedMode serializerMode,
             out SerializedMode deserializerMode,
-            out string runMode) where T : Delegate
+            out string runMode)
+        {
+            var udfWrapperData = GetUdfWrapperDataFromStream(
```

Comment:
Looks like I missed `var` usage in this file. :) Can you fix `udfWrapperData`, `nodeIndex`, `udfIndex` in this file? Thanks!